spawn_sdk/spawn_sdk/lib/system/spawn_system.ex

```diff
@@ -70,7 +70,11 @@ defmodule SpawnSdk.System.SpawnSystem do
   @impl SpawnSdk.System
   def spawn_actor(actor_name, spawn_actor_opts) do
     system = Keyword.get(spawn_actor_opts, :system, nil)
-    actor_mod = Keyword.get(spawn_actor_opts, :actor, %{})
+    actor_mod = get_actor_module(spawn_actor_opts)
+
+    if is_nil(actor_mod) do
+      raise "Invalid Actor reference, actor not found registered in this instance"
+    end
 
     if not actor_mod.__meta__(:abstract) do
```

Comment:
shouldn't this be delegated somehow to proxy? so we can spawn_actors just by referencing by name without the need to have it registered in the system we are working

Reply:
Although there is no constraint defined in the proxy after the last changes made to it, today it is even more necessary to have this type of guarantee in the sdks